core/XH.js

```diff
@@ -309,6 +309,8 @@ class XHClass {
     track(opts)                 {return this.trackService.track(opts)}
     fetch(opts)                 {return this.fetchService.fetch(opts)}
     fetchJson(opts)             {return this.fetchService.fetchJson(opts)}
+    postJson(opts)              {return this.fetchService.postJson(opts)}
```

Comment:
Minor nit, but I would argue that we don't need these aliases (and we could probably do without the fetch()) alias as well).  The idea here, is just to provide shortcuts for the most frequently used service methods.   FetchService is a public API, and it is totally valid to access it via XH.fetchService.postJson(),

Reply:
I like these aliases.  I would argue these are some of the most frequently used service methods.